docs/components/HomeHero.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <main class="d-container">
+    <!-- Hero card -->
+    <div class="relative sm:rounded-2xl sm:overflow-hidden">
+      <!-- Background -->
+      <div class="absolute inset-0">
+        <img
```

Comment:
There is flicker in background loading not sure related to this or not

Reply:
There was a mistake with the initial SVG export, I think it will fix the problem.